pkgs/top-level/python-packages.nix

```
@@ -6383,7 +6383,7 @@ in {
 
   maildir-deduplicate = callPackage ../development/python-modules/maildir-deduplicate { };
 
-  mps-youtube = callPackage ../development/python-modules/mps-youtube { };
+  mps-youtube = throw "mps-youtube has moved to all-packages, and is no longer available as a python module.";
```

would love to do this, but we really need an `removed-python.nix`.
I don't think this should be a `throw` though, `mps-youtube` and `python3Packages.mps-youtube` should be equivalent, and I have no reason to believe anyone used it as a module.

I would suggest the following
```suggestion
  mps-youtube = pkgs.mps-youtube;
```

but we really shouldn't be having aliases in `python-packages.nix`, we'd have the same situation that created `aliases.nix` in the first place.

---

you would need to do something like ` mps-youtube = pkgs.mps-youtube.override { inherit python;};` otherwisei you'll get different interpreter versions.

---

Right, so no alias or warning at all. Maybe not the best UI, but easiest for Nixpkgs maintainers. I like that.